decorify/profiling.py

```diff
@@ -148,3 +150,53 @@ def inner(*args, **kwargs):
         return res
 
     return inner
+
+def track_leaks(__func__: Callable[..., Any]) -> Callable[..., tuple[Any, int]]:
+    """Decorator for finding memory leaks. 
+    It utilizes Python's gc module to track objects before and after function call.
+    The decorator returns a tuple of function's output and number of leaked objects.
+
+    Args:
+        __func__ (Callable[..., Any], optional): The function to wrap. Defaults to None.
+
+    Returns:
+        Callable[..., tuple[Any, int]]: Wrapped function that returns a tuple of function's output and number of leaked objects.
```

Comment:
Could you match style used in rest of repository for consistency?

Reply:
Well your style does break IDE docstring parsing on my end, but sure